scripts/check_pytests.py

```diff
@@ -23,9 +23,10 @@
 # is covered here as well since those tests are not run on NayDuck any more.
 HELPER_SCRIPTS = [
     'delete_remote_nodes.py',
+    'loadtest/*',
     'mocknet/*',
+    'shardnet/*',
     'stress/hundred_nodes/*',
-    'loadtest/*',
```

Comment:
I’m beginning to think we need to move all of those into a different directory. pytest/tools perhaps?

Reply:
+1 for 'python tools' directory somewhere